content/04.results.md

```diff
@@ -2,8 +2,23 @@
 
 ### Comparing bioRxiv to PubMed Central
 
-#### Global View
-1. Create a treemap visualization of top X terms that are different between bioRxiv and PubMed Central (based on odds ratio)
+#### Global Comparison of bioRxiv and PubMed Central 
+
+![
+BioRxiv is more focused on biological discoveries rather than disease treatments and clinical trials.
+The plot on the left (A) is a point range plot of the odds ratio with respect to bioRxiv.

```

Comment:
Are there confidence intervals on this? I can't tell if they are there but so small because of the total sample size that they don't appear, or if they are not present.

Reply:
There are confidence intervals; however, they are quite tiny (order of second or third decimal place).